src/Button.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled';
+
+interface IProps {
+  secondary?: boolean
+}
+
+export const Button = styled.button<IProps>`
```

Comment:
I wonder if we end up with 2 layers to this library: things are depend on TRC (<sheetContainer>, etc) and things that are truly generic (like Button, HorizontalList). Maybe we use the generic ones in our other projects (where we don't have the TRC APIs)

Reply:
I see what you are saying. We could play it out and see what happens, and if the need arises we can split these type of components in a more generic library.

Reply:
What if for starters, we kept in same library (as that definitely simplifies the mechanics) but split them by directory? Had a "common" directory for the generic controls.  If that doesn't work, ok.